http_service/cmd/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/go-kit/kit/endpoint"
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	httpuserservice "github.com/jcromanu/final_project/http_service/pkg/http_user_service"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+func main() {
+
+	var (
+		logger      log.Logger
+		userGRPC    *grpc.ClientConn
+		grpcErr     error
+		middlewares = []endpoint.Middleware{}
+		opts        []grpc.DialOption
+	)
+	opts = append(opts, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	httpAddr := flag.String("http.addr", ":8081", "HTTP listen address")
+
+	logger = log.NewLogfmtLogger(os.Stderr)
+	logger = log.With(logger, "ts", log.DefaultTimestampUTC)
+	logger = log.With(logger, "caller", log.DefaultCaller)
+	addr := "localhost:8080"
```

Comment:
could it change in execution o cabn be in a const, even in a enviroment variable.

Reply:
ok , I change it to env variables